source/docs/3/importing_spreadsheets_csv.rst

```diff
@@ -5,44 +5,65 @@ Many times the GIS data comes in a table or an Excel spreadsheet. Also, if you h
+     :align: center
+
+2. QGIS comes with a unified data manager that allows you to load all the various supported data formats. Click the :guilabel:`Open Data Source Manager` button on the :guilabel:`Data Source Toolbar`. You may also use :kbd:`Ctrl + L` keyboard shortcut.
+
+  .. image:: /static/3/importing_spreadsheets_csv/images/02.png
+     :align: center
+
+3. In the :guilabel:`Data Source Manager` dialog box, switch to the :guilabel:`Delimited Text` tab. Click the :guilabel:`...` button next to the :guilabel:`File name`.
+
+  .. image:: /static/3/importing_spreadsheets_csv/images/03.png
+     :align: center
+
+4. Depending upon the operating system, you may or may not view the file at the downloaded location. In File formats, switch to ``All files (* *.*)`` to view the *tsv* file. 
 
-  .. image:: /static/3/importing_spreadsheets_csv/images/1.png
+  .. image:: /static/3/importing_spreadsheets_csv/images/04.png
      :align: center
 
-2. QGIS3 comes with a unified data manager that allows you to load all the various supported data format. Click the :guilabel:`Open Data Source Manager` button on the :guilabel:`Data Source Toolbar`. You can also use :kbd:`Ctrl + L` keyboard shortcut.
+5. Now you will see the downloaded file. Select that and click :guilabel:`Open`. 
 
-  .. image:: /static/3/importing_spreadsheets_csv/images/2.png
+  .. image:: /static/3/importing_spreadsheets_csv/images/05.png
      :align: center
 
-3. Switch to the :guilabel:`Delimited Text` tab. Click the :guilabel:`...` button next to :guilabel:`File name` and specify the path to the text file you downloaded. In the :guilabel:`File format` section, select :guilabel:`Custom delimiters` and check :guilabel:`Tab`. The :guilabel:`Geometry definition` secction will be auto-populated if it finds a suitable X and Y coordinate fields. In our case they are ``LONGITUDE`` and ``LATITUDE``. You may change it if the import selects the wrong fields. You can leave the :guilabel:`Geometry CRS` to the default ``EPSG:4326 WGS84`` CRS. If your file contained coordinates in a different CRS, you can select the appropriate CRS here. Click :guilabel:`OK`.
+6. In the :guilabel:`Data Source Manager` dialog box, the path to file will be available in :guilabel:`File Name`. Change the :guilabel:`Layer name` to ``1900_2000_earthquake``. In the :guilabel:`File format` section, select :guilabel:`Custom delimiters` and check :guilabel:`Tab`. In the :guilabel:`Geometry definition` section, choose :guilabel:`Point coordinates` by default import will auto-populate if it finds a suitable :guilabel:`X field` and :guilabel:`Y field`. In our case, they are ``Longitude`` and ``Latitude``. You may change it if the import selects the wrong fields. You can leave the :guilabel:`Geometry CRS` to the default ``EPSG:4326 - WGS 84`` CRS. If your file contained coordinates in a different CRS, you could select the appropriate CRS here. Click :guilabel:`Add`.
```

Comment:
choose :guilabel:`Point coordinates`. By default the :guilabel:`X field` and :guilabel:`Y field` values will be auto-populated if it finds a suitably named field in the input.

Reply:
changed.